packages/search-ui-app-search-connector/src/AppSearchAPIConnector.js

```diff
@@ -53,7 +53,8 @@ class AppSearchAPIConnector {
     beforeAutocompleteResultsCall = (queryOptions, next) => next(queryOptions),
     beforeAutocompleteSuggestionsCall = (queryOptions, next) =>
       next(queryOptions),
-    endpointBase = ""
+    endpointBase = "",
+    ...rest
```

Comment:
This is a very clean approach BUT means that any other new parameters automatically become "breaking changes".  In this case _probably_ fine since we control the App Search API as well, but wanted to highlight that.

Reply:
Good point. Hmmm, I think it's probably going to be fine also, but that isn't something I had considered. Thank you for mentioning it.